src/rules/matchDescription.js

```diff
@@ -47,18 +52,36 @@ export default iterateJsdoc(({
   tags.some((tag) => {
     const description = _.trimStart(tag.description, '- ');
 
-    return validateDescription(description, tag.tag);
+    return validateDescription(description, tag);
   });
 }, {
+  contextDefaults: true,
   meta: {
     schema: [
       {
         additionalProperties: false,
         properties: {
+          contexts: {
+            oneOf: [
+              {
+                items: {
+                  type: 'string'
+                },
+                type: 'array'
+              },
+              {
+                type: 'string'
+              }
```

Comment:
I prefer allowing only `array` to keep it simple.

Reply:
I prefer accepting string for simplicity to the user. :) @gajus ?

Reply:
Please only accept `array`. Multiple APIs create ambiguity about how to supply multiple parameters.

Reply:
Ok, done.